src/components/hero/hero.module.scss

```diff
@@ -0,0 +1,90 @@
+@use "breakpoints";
+@import "variables";
+
+.hero {
+  grid-column: 1 / -1 !important;
+
+  width: 100%;
+  height: 30rem;
+  display: inherit;
+  background-repeat: no-repeat;
+  background-size: cover;
+
+  grid-template-columns: inherit;
+  grid-template-rows: inherit;
+
+  & > * {
+    grid-column: 2;
+  }
+
+  @include breakpoints.respond-above(m) {
+    height: 40rem;
+  }
+
+  .box {
+    color: $color-white;
+    background-color: rgba($color-black, 80%);
+    width: 20rem;
+    height: 15rem;
+    margin-top: var(--spacing-xl);
+    padding: var(--spacing-m);
```

Comment:
You've used scss variables elsewhere, but css variables here. Is there a reason for it?

Reply:
Not really, constant context switching messes with my thinking. I'll fix these.